frame/election-provider-multi-phase/src/unsigned.rs

```diff
@@ -143,48 +153,58 @@ impl<T: Config> Pallet<T> {
 		);
 
 		let call = restore_solution::<T>()
-			.and_then(|call| {
-				// ensure the cached call is still current before submitting
-				if let Call::submit_unsigned(solution, _) = &call {
+		.and_then(|call| {
+			// ensure the cached call is still current before submitting
+			if let Call::submit_unsigned(solution, _) = &call {
 					// prevent errors arising from state changes in a forkful chain
```

Comment:
Nit: looks like you dedented the overall `.and_then` block, but not its contents. Could use more consistent spacing.